aws/asg-ebs/main.tf

```diff
@@ -39,6 +39,10 @@ resource "aws_launch_template" "asg_lt" {
     }
   }
 
+  credit_specification {
+    cpu_credits = var.aws_instance["cpu_credits"]
```

Comment:
I don't believe this will work for backwards compatibility. For consumers that specify an aws_instance without `cpu_credits`, this will produce `cpu_credits = null` or maybe even cause an error. Either way, it will break existing code. I think you either need to bump the major version when this is released, or use a new variable for this value. I would prefer the latter.

Reply:
Agreed.  Changed to use a new variable with a default value of 'null'.